### PR TITLE
Create 2020-01-06.md

### PR DESCRIPTION
**Pull request by mmarchini**

### CODE REVIEW

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by Qard — APPROVED** [review]

File: wg-meetings/2020-01-06.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Node.js  Diagnostics WorkGroup Meeting 2021-01-06
+
+## Links
+
+* **Recording**:  https://github.com/nodejs/diagnostics/issues/460
```
Comment (gireeshpunathil) [inline wg-meetings/2020-01-06.md:5]:
```suggestion
* **Recording**:  https://www.youtube.com/watch?v=rPiD0OchmpA
```

File: wg-meetings/2020-01-06.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* **Minutes Google Doc**: $MINUTES_DOC$
```
Comment (gireeshpunathil) [inline wg-meetings/2020-01-06.md:6]:
```suggestion
* **Minutes Google Doc**: https://docs.google.com/document/d/1qCrsUxc5sqXczKxVE1tWmf_LR2Ci0llNQ-HzDTTMnAU/edit
```

**Review by gireeshpunathil — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by Qard — APPROVED** [review]

**Comment by gireeshpunathil** [comment]
landed in 49edf4a7fc6c03ec071c6b245996d618ff75a529

**Pull request closed without merging**